purchase_landed_cost/models/stock_picking.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013 Joaquín Gutierrez
+# Copyright 2014-2016 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.multi
+    def action_open_landed_cost(self):
+        self.ensure_one()
+        line_obj = self.env['purchase.cost.distribution.line']
+        lines = line_obj.search([('picking_id', '=', self.id)])
+        if lines:
```

Comment:
Create new common method to pass lines and use in both models or change same commented above.

Reply:
Done